src/dmd/cppmangle.d

```diff
@@ -65,16 +65,21 @@ const(char)* cppTypeInfoMangleItanium(Dsymbol s)
 }
 
 /******************************
-* Determine if a symbol is the 'primary' destructor, that is, the most-aggregate destructor (the one that is defined as __xdtor)
-*/
-package bool isPrimaryDtor(Dsymbol sym)
+ * Determine if sym is the 'primary' destructor, that is,
+ * the most-aggregate destructor (the one that is defined as __xdtor)
+ * Params:
+ *      sym = Dsymbol
+ * Returns:
+ *      true if sym is the primary destructor for an aggregate
+ */
+bool isPrimaryDtor(const Dsymbol sym)
```

Comment:
I'd like to know your justification for making this `public` rather than `package` so I can apply such consideration in future PR reviews.  I would have expected that we'd prefer `package` over `public`;  that is, we restrict access as much as possible until justification is provided to expose a symbol more broadly.  In other words, the burden of proof is on the desire to expose access, not restrict access.  It's always possible to expose access to a symbol, but once it's exposed, it's a breaking change to restrict access.

Reply:
We don't use packages in dmd.

Reply:
Actually we do.  `dmd` is a package, and if we don't take care about how much we expose publicly, we'll just be creating unnecessary restrictions and maintenance for ourselves.